galaxyui/src/app/authors/detail/author-detail.component.html

```diff
@@ -172,7 +199,7 @@
     <div class="row repository-pagination" *ngIf="!authorNotFound">
         <div class="col-sm-12">
             <div class="pagination"
-                *ngIf="items && items.length && paginationConfig.totalItems > paginationConfig.pageSize">
+                *ngIf="repositories && repositories.length && paginationConfig.totalItems > paginationConfig.pageSize">
```

Comment:
This looks like we'll only include pagination if there are repositories, but we want to show pagination if there are repositories AND/OR collections, right?